server/src/test/java/io/unitycatalog/server/base/catalog/BaseCatalogCRUDTest.java

```diff
@@ -39,7 +41,11 @@ protected void assertCatalogNotExists(List<CatalogInfo> catalogList, String name
     public void testCatalogCRUD() throws ApiException {
         // Create a catalog
         System.out.println("Testing create catalog..");
-        CatalogInfo catalogInfo = catalogOperations.createCatalog(CATALOG_NAME, COMMENT);
+        CreateCatalog createCatalog = new CreateCatalog()
+                .name(CATALOG_NAME)
+                .comment(COMMENT)
+                .properties(PROPERTIES);
+        CatalogInfo catalogInfo = catalogOperations.createCatalog(createCatalog);
         assertCatalog(catalogInfo, CATALOG_NAME, COMMENT);
```

Comment:
where are you testing that the properties has been set?

Reply:
I'm not because currently because the catalog and schema CLI do not support properties, hence, properties from CreateCatalog/CreateSchema are not passed [here](https://github.com/unitycatalog/unitycatalog/pull/89/files#diff-14d66fe25cca8e18a15e074ac909c939e105c7a8045579eded058006cef2455eR27). I've added todos so that we can make changes to the CLI to accept properties for catalog and schema, post which we can add those validations here.